client/src/components/Footer.css

```diff
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+
+footer {
+    padding: 71px 78px; 
+    background-color: #000000;
+    color: #ffffff;
+    font-family: 'Roboto', sans-serif;
+    font-size: 20px;
+    font-weight: 400;
+}
+
+.flex-container {
+    display: flex;
+    justify-content: space-between;
+}
+
+.footer-disclaimer {
+    width: 824px;
+    margin-bottom: 80px;
+}
+
+.bold-font {
+    font-weight: bold;
+}
+
+.medium-font {
+    font-weight: 500;
+}
+
+.organisation-links-container {
+    display: flex;
+    flex-direction: column;
+    gap: 40px
+}
+
+.organisation-container {
+    width: 533px;
+    margin: 16px;
+}
+
+.external-links-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.link {
```

Comment:
This css should be with the `ExternalLink` component - try creating an `ExternalLink.css` file and import it into `ExternalLink.js`

Reply:
Same with `.link-icon` and `.bold-font` which is only used in `OrganisationalLink`.

By housing the CSS with the relevant component, you make them more reusable. With this CSS setup, these components can only be used in the Footer, otherwise they break.